.cirrus.star

```diff
@@ -1,4 +1,430 @@
+# Check Starlark specs: https://github.com/bazelbuild/starlark/blob/master/spec.md
+# Cirrus
+load("cirrus", "yaml", "fs")
```

Comment:
Why is it needed? If I see correctly none of yaml or fs functions are used.

Reply:
Part of the template in the migration tool but indeed those helper functions are not required in this pipeine.